packages/api/src/external/commonwell/admin/find-patient-duplicates.ts

```diff
@@ -59,16 +59,14 @@ export type DuplicatedPersons = {
 /**
  * Checks for duplicated persons in CommonWell.
  */
-export async function findDuplicatedPersons(cxId?: string): Promise<DuplicatedPersons> {
+export async function findDuplicatedPersons(cxId: string): Promise<DuplicatedPersons> {
```

Comment:
Updated this because this was too destructive, from the early days when we didn't have many patients. It would be too expensive to this for all customers.